packages/bruno-app/src/components/Sidebar/NewRequest/index.js

```diff
@@ -58,6 +60,12 @@ const NewRequest = ({ collection, item, isEphemeral, onClose }) => {
           name: 'requestName',
           message: `The request names - collection and folder is reserved in bruno`,
           test: (value) => {
+            if (isWindowsOS() && variableNameRegex.test(value) === false) {
```

Comment:
The variable name regex (`/^[\w-.]*$/`) is too strict. And does not allow Chinese characters etc.

Reply:
Hey, I just updated the regex, tell me if its ok for you.